### PR TITLE
switches from onMouseEnter to onMouseMove, works more reliably

### PR DESCRIPTION
**Pull request by Raemon**

### CODE REVIEW

**Comment by Discordius** [comment]
I don't think this is a good idea. This will fire hundreds of events in a very short time, which I expect can easily cause lags and other bugs. See here for a demonstration: 

https://www.w3schools.com/jquery/tryit.asp?filename=tryjquery_event_mouseenter_mouseover

You might want to try onMouseOver, which will still trigger more often, but not as ridiculously often as the `onMouseMove`

File: packages/lesswrong/components/common/withHover.jsx [hunk 1]
```diff
@@ -4,7 +4,7 @@ const withHover = (WrappedComponent) => {
   return class HoverableComponent extends Component {
     state = { hover: false, anchorEl: null }
 
-    handleMouseEnter = (event) => {
+    handleMouseMove = (event) => {
       this.setState({ hover: true, anchorEl: event.currentTarget})
```
Comment (jimrandomh) [inline packages/lesswrong/components/common/withHover.jsx]: Is `anchorEl` used by any of this element's children? It's doing something very counterintuitive; it's the deepest-in-tree element that the mouse is over, which is usually not what you want.

**Comment by jimrandomh** [comment]
When is `onMouseEnter` unreliable?

File: packages/lesswrong/components/common/withHover.jsx (hunk 1 above)
Comment (Discordius) [reply packages/lesswrong/components/common/withHover.jsx]: That seems true, my guess is we actually want to have a reference to the div that is introduced by the `withHover` component, which we can be sure doesn't change.

**Comment by Discordius** [comment]
In my experience it has been unreliable when you have the dom move under the mouse. I.e. you resolve a sidebar item, and then the lower ones move up, but that alone doesn't trigger the onMouseEnter.

**Comment by Raemon** [comment]
Updated to use onMouseOver. This appears to work fine with event.currentTarget (although triggering on entering different elements, it still is using the root element for current target)

**Pull request merged**